comp/core/gui/guiimpl/gui.go

```diff
@@ -240,49 +229,57 @@ func serveAssets(w http.ResponseWriter, req *http.Request) {
 	w.Write(data)
 }
 
+func (g *gui) getAccessToken(w http.ResponseWriter, r *http.Request) {
+
+	intentToken := r.URL.Query().Get("intent")
+	// authToken is present in the query when the GUI is opened from the CLI
```

Comment:
```suggestion
	// authToken is not present in the query when the GUI is opened from the CLI
```

Reply:
Done in 269022d16e493c88bdcdb13579f4cf583e2e6031.